scripts/treeDataFromFileTree.ts

```diff
@@ -0,0 +1,78 @@
+import { promises as fs, Stats } from 'fs';
+import path from 'path';
+import * as yargs from 'yargs';
+
+interface Node {
+  hierarchy: string[];
+  size: number;
+  updatedAt: string;
+}
+
+const getFileNode = (filePath: string, parentHierarchy: string[], stats: Stats): Node => {
+  const basename = path.basename(filePath);
+
+  return {
+    hierarchy: [...parentHierarchy, basename],
+    size: stats.size,
+    updatedAt: stats.mtime.toISOString(),
+  };
+};
+
+const getSubTree = async (nodePath: string, parentHierarchy: string[] = []) => {
+  const nodes: Node[] = [];
+  const stats = await fs.lstat(nodePath);
+
+  if (stats.isDirectory()) {
+    const children = await fs.readdir(nodePath);
+    const directory = path.basename(nodePath);
```

Comment:
We need to ignore `node_modules` and `build`. If I run `yarn dataset:file-tree --path .` it passes through these folders.